types/vochain.go

```diff
@@ -90,8 +90,9 @@ type BlockParams struct {
 }
 
 type EvidenceParams struct {
```

Comment:
These are Vochain specific types, right? Why not moving them to vochain/types.go ?